.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,9 +2,12 @@
 
 [Please briefly describe what you have changed.]
 
+
+
 ## Additional context
 
-[If a screenshot would be useful (e.g. for UI changes), please provide one. If not, please remove this section]
```

Comment:
Could you please bring this line back?

Reply:
@knjk04 Hello, I did everything except this, pushed and tests passed, last step I add this line back and tests not passed. What should I do now?